pyface/qt/QtCore.py

```diff
@@ -33,14 +21,7 @@
     __version__ = QT_VERSION_STR
     __version_info__ = tuple(map(int, QT_VERSION_STR.split(".")))
 
-
-elif qt_api == "pyside2":
+else:
```

Comment:
I think it is OK to do this, since Pyside2 is the blessed Python Qt wrapper provided by the Qt Company. If a new wrapper cam along it would likely follow the Pyside conventions more than the PyQt conventions.